xml/vt_scenarios.xml

```diff
@@ -0,0 +1,198 @@
+	 xmlns:xlink="http://www.w3.org/1999/xlink" 
+	 version="5.0" xml:id="cha-virtualization-scenarios">
+ <!-- I/O virtualization -->
+ <title>Virtualization Scenarios</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+   </dm:bugtracker>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Virtualization design provides several useful capabilities to your
+    organization: more efficient hardware use, support for legacy software,
+    operating system isolation, live migrations, disaster recovery, and 
+    load balancing. 
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec-virtualization-scenarios-capabilities">
+  <title>Server Consolidation</title>
+  <para>
+   Many servers can be replaced by one big physical
+   server, so hardware is consolidated, and guest operating systems are
+   converted to virtual machines. This also supports running legacy
+   software on new hardware.
+   <!-- reduce cost -->
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Better usage of resources which were not running at 100%
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Fewer servers locations needed
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Compute resources more efficiently: multiple workloads on the same server
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Simplification of the data center infrastructure
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Moving workload is easier to another host, this will avoid service downtime 
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Virtual Machine provisioning
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Multiple guest operating system will be able to run on 1 host
+    </para>
+   </listitem>
+  </itemizedlist>
+  <important>
+   <para>The server consolidation scenario needs special attention.</para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Maintenance windows should be carefully planned
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Storage is a key point to be able to support migration and growing disk usage
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Server capabilities must be checked to support the new workload
+     </para>
+    </listitem>
+   </itemizedlist>
+  </important>
+ </sect1>
+ <sect1>
+  <title>Isolation</title>
+  <para>
+   A guest operating system can be fully isolated from the host
+   running it. So if the virtual machine is corrupted, the host system is
+   not harmed. It also does not affect any other virtual machines. No data
+   is shared between VM.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Secure Boot can be used for VM
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     KSM should be avoided
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     CPU can be assigned to a VM, and HT should be disabled to avoid security issue
```

Comment:
```suggestion
     Individual CPU cores can be assigned to VMs.
    </para>
    </listitem>
    <listitem>
    <para>
     Hyper-threading (HT) should be disabled to avoid potential security issues
```